### ISSUE TITLE
MVP: Quality, user-focused support content (policies, FAQ, help documentation, service information)

### ISSUE THREAD

**Issue by heidisb** [mlibrary/umrdr#49]
I have been working with @bnhowell, Amy, Jen and Jake C. on Help pages. The content so far is based on a teardown of the [draft policy](https://docs.google.com/a/umich.edu/document/d/1Qr_3zGPvLFhlgCcz-DpgJ-3EpCj0Lii96DOv5gerFa0/edit?usp=sharing).

[The copy is in a Google Doc](https://docs.google.com/a/umich.edu/document/d/1ipy_teq-TbssePRAExdIwweF7nvul4uNULfiIZix2jM/edit?usp=sharing).
And I also [mocked up the IA using copy available](https://docs.google.com/a/umich.edu/folderview?id=0B3qenPHezcYTcmVsZ3ZoYnNoRnc&usp=drive_web) so far and placeholders.

I am happy to help teardown content and polish it for the web, but know there are groups with the proper knowledge to work on the writing. **We need copy for:**
- About Us (Intro and Why deposit)
- How to Upload
- Support for Depositors 
- Use Downloaded Data

Additional notes:
- About Us page is what we will want to link to from the big call to action on the homepage.
- Policies and Terms of Use page (draft doc, not mocked up yet), should be linked in the footer. 
- Possibly link Contact Us in the footer, as well.

**Comment by heidisb** [comment]
Jake C. filled in the copy and I updated the mock-ups on Friday. We passed off the mockups and copy to Colin F. this morning to put into production.

Notified stakeholders that if they want the full Policy & Terms of Use page live for Feb. 28th, they need to get on whatever cleanup needs to happen to the draft policy so we can turn it into a webpage. My suggestion has been putting a link to it in the footer, which seems well received.
